### PR TITLE
Fix scroll event listener to new standard

### PR DESCRIPTION
**Pull request by Tyler-Maclachlan**
I messed up my fork for PR #170, so I created a new fork and created a new PR.

@Thomaash I see what you meant, I have changed my code accordingly, it should be much cleaner now. :+1:

### CODE REVIEW

File: lib/network/modules/Canvas.js [hunk 1]
```diff
@@ -249,8 +249,7 @@ class Canvas {
     this.hammer.on('pinch',     (event) => {this.body.eventListeners.onPinch(event)});
 
     // TODO: neatly cleanup these handlers when re-creating the Canvas, IF these are done with hammer, event.stopPropagation will not work?
-    this.frame.canvas.addEventListener('mousewheel',     (event) => {this.body.eventListeners.onMouseWheel(event)});
-    this.frame.canvas.addEventListener('DOMMouseScroll', (event) => {this.body.eventListeners.onMouseWheel(event)});
+    this.frame.canvas.addEventListener('wheel',     (event) => {this.body.eventListeners.onMouseWheel(event)});
```
Comment (Thomaash) [inline lib/network/modules/Canvas.js]: Just a cosmetic change that really annoys me. Could you leave just one space here, please?

Comment (Tyler-Maclachlan) [reply lib/network/modules/Canvas.js]: Didn't notice it added extra spaces. They're removed.

**Comment by Tyler-Maclachlan** [comment]
Tested on Fedora Linux 29
Chrome Version 78.0.3904.70
Firefox Version 69.0.1 (64-bit)

**Comment by Thomaash** [comment]
Okay, so I'm adding Windows 10 (64 bit) tests:
- [x] Google Chrome 78.0.3904.87,
- [x] Opera 64.0.3417.83,
- [x] Vivaldi 2.9.1705.38,
- [x] Firefox 70.0.1,
- [x] Microsoft Edge 44.18362.449.0,
- [x] Internet Explorer 11.418.18362.0.

**Pull request merged**